### PR TITLE
Import enumification-helpers from monodroid.

### PR DESCRIPTION
**Pull request by atsushieno**
...as of ade615d

There are slight changes in Makefile and README from monodroid.

### CODE REVIEW

File: build-tools/enumification-helpers/README [hunk 1]
```diff
@@ -0,0 +1,256 @@
+
+This directory contains a couple of "helper" tools for API int-to-enum conversion.
```
Comment (jonpryor) [inline build-tools/enumification-helpers/README]: Should *this* file go into the top-level `Documentation` folder? :-)

Comment (jonpryor) [inline build-tools/enumification-helpers/README]: Even if it s wouldn't be moved, we should reformat this as Markdown.

Comment (atsushieno) [inline build-tools/enumification-helpers/README]: This documentation is not really in quality. I'd rather "completely remove" this if it were to go there and supposed to "become for everyone".

Comment (atsushieno) [inline build-tools/enumification-helpers/README]: Regarding "making changes before porting", we have two options: keep consistent with monodroid/oss-history or do not import

Comment (jonpryor) [inline build-tools/enumification-helpers/README]: Then I rescind my suggestion. :-)

**Pull request merged**